SU-HSDO/suhumsci
# fix(shs-5060): fix H2s around titles in bookshelf/publication cards

# READY FOR REVIEW

## Summary
This PR adds a check in the vertical card template and the horizontal card template for if a heading tag has been set in the view, or if it needs to be manually added in the template.  

## Steps to Test
1. Visit publications default view in a local instance, such as: https://hs-traditional.suhumsci.loc/default-views/publications
2. Note that publications no longer has the duplicated heading tags (`<h2></h2><h3>Title</h3>`). 
3. Verify that other vertical or horizontal cards are not affected (they EITHER have their custom view's heading tag or the `<h2>` tag coming from the template. 

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
